crates/telio-crypto/src/lib.rs

```diff
@@ -130,7 +132,7 @@ impl SecretKey {
     /// assert_ne!(pub_key_a, pub_key_b);
     /// ```
     pub fn public(&self) -> PublicKey {
-        crypto_box::SecretKey::from(self.0).public_key().into()
+        PublicKey(MontgomeryPoint::mul_base(&Scalar::from_bytes_mod_order(clamp_integer(self.0))).0)
```

Comment:
This is a very dense line of code. Can we make it more understandable? Perhaps with a comment or split it into stages with descriptive variable names?

Reply:
@mathiaspeters well, I changed it while thinking about getting rid of this crate completely, but I guess in the current state it will be better to just leave the old version here